ui/lobby/src/setup.ts

```diff
@@ -919,9 +919,10 @@ export default class Setup {
     const setAnonOptions = () => {
       const isAnon = $form.data('anon');
       if (isAnon) {
-        setBaseDefaultOptions(); //defult to chess, real time, blitz clock
+        setBaseDefaultOptions(); //default to chess, real time, blitz clock
         $casual.trigger('click');
-        $opponentInput.val('lobby'); //default to lobby
```

Comment:
I think we still want to make the default form have bot as opponent for anon users, rather than lobby when not challenging a bot directly.